interface/main/messages/trusted-messages.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+/**
+ * trusted-messages.php displays the GUI and handles the interactions with the backend ajax processor for sending
+ * messages and file attachments to Trusted email addresses using the Direct protocol.
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2022 Discover and Change <snielson@discoverandchange.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Core\Header;
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\OeUI\OemrUI;
+use OpenEMR\Services\PatientService;
+
+require_once("../../globals.php");
+
+$message = '';
+if (isset($_REQUEST['message_code'])) {
+    $message_code = $_REQUEST['message_code'] ?? null;
+    switch ($message_code) {
+        case 'success':
+            $message = xlt("Trusted message was sent");
+            break;
+    }
+}
+
+// check if we have a selected patient already
+$pid = "";
+$patientName = "";
+if (!empty($_SESSION['pid'])) {
+    $patientService = new PatientService();
+    $patientArray = $patientService->findByPid($_SESSION['pid']);
+    if (!empty($patientArray)) {
+        $pid = $_SESSION['pid'];
+        // if things are empty this ends up being blank.
+        $patientName = trim(($patientArray['fname'] ?? '') . ' ' . ($patientArray['lname'] ?? ''));
+    }
+}
+
+if ($GLOBALS['medex_enable'] == '1') {
+    if ($_REQUEST['SMS_bot']) {
+        $result = $MedEx->login('');
+        $MedEx->display->SMS_bot($result);
+        exit();
+    }
+    $logged_in = $MedEx->login();
+} else {
+    $logged_in = null;
+}
```

Comment:
is this block supposed to be here? (just not seeing the $MedEx anywhere above)

Reply:
Nope, I originally copied over the messages navigation which uses that code block.  However, went a different direction and so this is good to be removed.  Good catch!

Reply:
yep, ole eagle eye!